docs/components/tfidf_ranking.rst

```diff
@@ -125,15 +137,14 @@ and is built by the following steps:
 #. Unpack and extract the articles with `WikiExtractor`_
    (with ``--json``, ``--no-templates``, ``--filter_disambig_pages``
    options)
-#. Build a database with the help of `DrQA
-   script <https://github.com/facebookresearch/DrQA/blob/master/scripts/retriever/build_db.py>`__.
+#. Build a database during :ref:`ranker_training`.
 
 enwiki_tfidf_matrix.npz
 -------------------------
 
 **enwiki_tfidf_matrix.npz** is a full Wikipedia tf-idf matrix of
 size **hash_size x number of documents** which is
-**2**24 x 5159530**. This matrix is built with
+**2**24 x 5180368**. This matrix is built with
```

Comment:
Maybe replace __2**24__ with __2²⁴__? I do not insist but it looks better in the doc in my opinion
![image](https://user-images.githubusercontent.com/5615053/46870348-02c83b00-ce37-11e8-8f81-876bb1ed840c.png)